src/WriteLockStrategy.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * This file is part of prooph/pdo-event-store.
+ * (c) 2016-2019 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2019 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Pdo;
+
+interface WriteLockStrategy
+{
+    public function getLock(string $name): bool;
+
+    public function releaseLock(string $name): bool;
```

Comment:
As the release failure is not dealt with (which I guess makes sense since the lock timeout would take care of adverse scenarios) I think this could be a void
```suggestion
    public function releaseLock(string $name): void;
```